tests_v2/unit/app/routes/datamart/test_land.py

```diff
@@ -103,14 +103,14 @@ async def test_get_tree_cover_loss_by_drivers_with_overrides(
         )
 
         response = await async_client.get(
-            "/v0/land/tree-cover-loss-by-driver?x-api-key={api_key}&geostore_id={geostore_id}&canopy_cover=30&dataset=umd_tree_cover_loss&version=v1.8&dataset=umd_tree_cover_density_2000&version=v1.6",
+            "/v0/land/tree_cover_loss_by_driver?x-api-key={api_key}&geostore_id={geostore_id}&canopy_cover=30&dataset=umd_tree_cover_loss&version=v1.8&dataset=umd_tree_cover_density_2000&version=v1.6",
```

Comment:
Is this supposed to be an f-string? I see `{api_key}` and `{geostore_id}` but no "f"

Reply:
That's a good question... not sure how it's passing!

Reply:
@dmannarino fixed, but it was passing because API key was already sent in the header, and the place geostore_id was being used was mocked. So I think the test is actually testing things.